src/plone/restapi/services/users/get.py

```diff
@@ -71,8 +71,9 @@ def _get_user(self, user_id):
     @staticmethod
     def _sort_users(users):
         users.sort(
-            key=lambda x: x is not None
-            and normalizeString(x.getProperty("fullname", ""))
+            key=lambda x: normalizeString(x.getProperty("fullname", ""))
+            if x is not None
```

Comment:
This must never happen if L97 was running before. I would skip the test here. To clarify what is expected we could add  a type hint and some doc-string to the function.

Reply:
Done, I added a docstring but not a type hint as I am not sure how to express "class instance but not None" and not sure if we can assert the actual class of the input.